packages/core/src/components/Status/Status.tsx

```diff
@@ -57,43 +71,23 @@ const useStyles = makeStyles<BackstageTheme>((theme) => ({
 
 export const StatusOK: FC<{}> = (props) => {
   const classes = useStyles(props);
-  return (
-    <span
-      className={classNames(classes.status, classes.ok)}
-      aria-label="Status OK"
-      {...props}
-    />
-  );
+  return <span className={classNames(classes.status, classes.ok)} {...props} />;
 };
 
 export const StatusWarning: FC<{}> = (props) => {
   const classes = useStyles(props);
   return (
-    <span
-      className={classNames(classes.status, classes.warning)}
-      aria-label="Status warning"
-      {...props}
-    />
+    <span className={classNames(classes.status, classes.warning)} {...props} />
   );
 };
 
 export const StatusError: FC<{}> = (props) => {
   const classes = useStyles(props);
   return (
-    <span
-      className={classNames(classes.status, classes.error)}
-      aria-label="Status error"
-      {...props}
-    />
+    <span className={classNames(classes.status, classes.error)} {...props} />
```

Comment:
Why remove the `aria-label`?

Reply:
Aria attributes on non-semantic elements (span, divs) are often omitted by assistive technology so the message was not communicated to a user. As @katz95 mentioned, `Status` component should be used with text - making it accessible without any extra metadata. Additionally, one should keep in mind that `aria-labels` should be translated when applying internationalization (extra complexity). I'm happy to revert that if it doesn't make sense!

Reply:
Oh, good point.